apps/mailrelay2/mailrelay2/dev/dev.yaml

```diff
@@ -46,3 +46,5 @@ spec:
       enableInboundTls: true
       enableInternalTls: true
       enableAuth: true
+
+    useWorkloadIdentity: true
```

Comment:
https://github.com/hmcts/sds-flux-config/pull/3685/files#diff-b951dc3dcd56820285f11be3a071462ddac0c31833272cf59d2f4fb8193a2213R17 also needs aadidentityname removing for both of these